### DISCUSSION TITLE
Maximised-size

### DISCUSSION THREAD

**Discussion by superpoincare** [tomasklaen/uosc#106] [Q&A]
Normally mpv would have three sizes at least. 

- The size of the video,
- maximised size which doesn't occupy the taskbar,
- full screen. 

But I don't see the second option with uosc.

Anyway to get the second?

**Comment by tomasklaen** [comment]
mpv has 2 props which you can cycle: `window-maximized` and `fullscreen`

The maximize button in uosc's top bar is already doing a `cycle window-maximized` command as you'd expect. The problem is that when window decorations are disabled (`border=no`), mpv seems to stop differentiating between them, and both behave like `cycle fullscreen`.

And I'm not aware about anything I can do about this on my end unfortunately.

Your options are either to use window decorations, or file an issue with mpv. Or maybe try asking on their irc channel beforehand.

Report back in case you find out about some fix :)
